src/components/search/TourItem.jsx

```diff
@@ -2,21 +2,41 @@ import { useMemo } from 'react'
 import { Link } from 'react-router-dom'
 import { Box, Typography } from '@mui/material'
 
-function TourItem({ attraction }) {
+function TourItem({ attraction, smallCard }) {
   const city = useMemo(
     () => (attraction?.address ? attraction.address.split(' ')[0] : ''),
     [attraction],
   )
 
   return (
     // 클릭시 여행지 디테일 페이지로 이동
-    <Link>
-      <Box className="h-29 group relative mb-6 flex overflow-hidden border border-solid border-gray-200 bg-white hover:border-blue-200 hover:shadow-lg hover:shadow-blue-100">
-        <Box className="h-[8rem] w-[8rem] overflow-hidden">
+    <Link to={`/tourDetail/${attraction.attractionId}`}>
+      <Box
+        className={
+          smallCard
```

Comment:
`smallCard`의 값에 상관없이 공통된 스타일은 묶어주시는게 좋을 것 같습니다.
```suggestion
className={`group flex overflow-hidden .... ${smallCard ? 'true-style' : 'false-style'}}
```